Bugfix/zenko 2153 run all func tests

dora-korpar opened the pull request:
Only 1 awssdk test was being run. This removes all `.only`s from the test suite

dora-korpar committed:
bf: ZENKO 2153 run all functional tests

dora-korpar committed:
[squash] rm other .only

bert-e commented:
# Hello dora-korpar,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/cloudserver/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `bugfix/ZENKO-2153-run-all-func-tests` into
`development/8.1`
* [w/8.2/bugfix/ZENKO-2153-run-all-func-tests](https://github.com/scality/cloudserver/compare/development/8.2...w/8.2/bugfix/ZENKO-2153-run-all-func-tests) will be merged into `development/8.2` (pull request #2181)



The following branches will **NOT** be impacted:

* `development/6.4`
* `development/7.4`
* `development/7.6`




*Follow* integration pull requests if you would like to be notified of
build statuses by email.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers





Peer approvals *must* include at least 1 approval from the following list:

* @rachedbenmustapha

* @dora-korpar

* @rahulreddy

dora-korpar committed:
[squash] unstring int

dora-korpar committed:
[drop] skip azure complete mpu tests

bert-e commented:
# History mismatch



Merge commit #2aba84390d91913f099346568c71a50ca1a43136 on the integration branch
`w/8.2/bugfix/ZENKO-2153-run-all-func-tests` is merging a branch which is neither the current
branch `bugfix/ZENKO-2153-run-all-func-tests` nor the development branch
`development/8.2`.

It is likely due to a rebase of the branch `bugfix/ZENKO-2153-run-all-func-tests` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**

alexandre-merle committed:
bgufix: S3C-3960: fix corner case multiple backend

in case of multiple backend, we were writing twice
'continue' in the socket, leading to econnreset from the client
side

bert-e committed:
Merge branch 'bugfix/S3C-3860-fix-corner-case-multiple-backend' into tmp/octopus/w/7.9/bugfix/S3C-3860-fix-corner-case-multiple-backend

alexandre-merle committed:
Merge remote-tracking branch 'origin/w/7.9/bugfix/S3C-3860-fix-corner-case-multiple-backend' into w/8.1/bugfix/S3C-3860-fix-corner-case-multiple-backend

alexandre-merle commented:
replaced by https://github.com/scality/cloudserver/pull/3336
